components/Hero/Hero.tsx

```diff
@@ -53,11 +46,15 @@ export function Hero() {
     <OuterWrapper ref={ref}>
       <Background
         initial={{ opacity: 0, translateX: "-10%", translateY: "10%" }}
-        animate={{ opacity: 0.15, translateX: "0%", translateY: "0%" }}
-        transition={{ duration: 0.5 }}
+        animate={{ opacity: 1, translateX: "0%", translateY: "0%" }}
+        transition={{ duration: 0.3 }}
       >
-        <LottieHeroAnimation play={isInView} animationData={animationData} />
+        <Video autoPlay loop muted playsInline>
```

Comment:
loading the lottie player library is expensive — it generates a lot a dom nodes and has to do a lot of parsing as mentioned before. lazy loading that stuff is fine for below the fold, but we can't lazy load this one in the hero because it constitutes the largest contentful paint. so instead we serve optimized video files, which are able to be preloaded by the browser